portfolio/src/main/webapp/script.js

```diff
@@ -38,12 +38,11 @@ function getData() {
 
     const dataListElement = document.getElementById('data-container');
     dataListElement.innerHTML = '';
-    dataListElement.appendChild(
-        createListElement('Comment 1: ' + data[0]));
-    dataListElement.appendChild(
-        createListElement('Comment 2: ' + data[1]));
-    dataListElement.appendChild(
-        createListElement('Comment 3: ' + data[2]));
+    data.forEach((item, index) => {
+      var index = index+1;
+      var item = item;
```

Comment:
Can you also inline these? Since you don't use index and item anywhere else

Reply:
yep!